src/terraform/README.md

```diff
@@ -1,5 +1,5 @@
 
-# Terraform, tflint, and TFGrunt (terraform)
+# Terraform, tflint, and Terragrunt (terraform)
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!